pkg/netpol/eval/internal/k8s/peer.go

```diff
@@ -13,18 +13,31 @@
 // limitations under the License.
 package k8s
 
+import (
+	"k8s.io/apimachinery/pkg/types"
+)
+
 // PeerType is a type to indicate the type of a Peer object (Pod or IP address)
 type PeerType int
 
 const (
 	PodType PeerType = iota
-	Iptype
+	IPBlockType
 )
 
 // Peer represents a k8s pod or an ip address
 type Peer struct {
-	PeerType  PeerType // PodType or Iptype
-	IP        string
+	PeerType  PeerType
+	IPBlock   *IPBlock // a set of intervals for ip addresses ranges
```

Comment:
Pod and IPBlock are mutually exclusive - that is, only one or the other can be set, not both>
If so, seems like a good case for Peer interface, implemented for either mode

Reply:
opened a separate issue #49